src/skia/Font.cpp

```diff
@@ -342,6 +350,59 @@ variationposition
         &SkFontArguments::VariationPosition::coordinateCount)
     ;
 
+py::class_<SkFontArguments::Palette> palette(
+    fontarguments, "Palette",
+    R"docstring(
+    A a palette to use and overrides for palette entries.
+    )docstring");
+
+py::class_<Override>(palette, "Override")
+    .def(py::init(
+        [] (uint16_t index, SkColor color) {
+            return Override({index, color});
+        }),
+        py::arg("index"), py::arg("color"))
+    .def("__repr__",
+        [] (const Override& self) {
+            return py::str("Override(index={}, color={})").format(
+                self.index, self.color);
+        })
+    .def_readwrite("index", &Override::index)
+    .def_readwrite("color", &Override::color)
+    ;
+
+py::bind_vector<std::vector<Override>>(palette, "Overrides");
+
+palette
+    .def(py::init(
+        [] (int index, const std::vector<Override>& overrides) {
+            SkFontArguments::Palette palette;
+            palette.index = index;
+            SetPaletteOverrides(palette, overrides);
+            return palette;
+        }),
+        py::keep_alive<1, 3>(),
+        py::arg("index"), py::arg("overrides"))
+    .def(py::init(
+        [] (int index) {
+            return SkFontArguments::Palette({index, nullptr, 0});
+        }),
+        py::arg("index"))
+    .def("__repr__",
+        [] (const SkFontArguments::Palette& self) {
+            return py::str("Palette(index={}, overrideCount={})").format(
+                self.index, self.overrideCount);
+        })
+    .def_property("overrides",
+        [] (const SkFontArguments::Palette& self) {
+            return std::vector<Override>(
+                self.overrides, self.overrides + self.overrideCount);
+        },
+        &SetPaletteOverrides)
+    .def_readwrite("index", &SkFontArguments::Palette::index)
```

Comment:
`PaletteIndex` might be a better choice of name, given font argument might have other indices.